### PR TITLE
feat(plugin): add limit quota header switch for limit-count

### PR DESCRIPTION
**Pull request by okaybase**
### What this PR does / why we need it:

Add limit quota header switch for limit-count plugin.
In some scenarios, such as some crawlers, service provider don't want to show limit quota info to the response header.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by tzssangglass** [comment]
why we need this? Is there a related issue?

**Comment by okaybase** [comment]
> why we need this? Is there a related issue?

In some scenarios, such as some crawlers, service provider don't want to show limit quota info to the response header.

File: apisix/plugins/limit-count.lua [hunk 1]
```diff
@@ -184,8 +185,10 @@ function _M.access(conf, ctx)
         return 500, {error_msg = "failed to limit count: " .. err}
     end
 
-    core.response.set_header("X-RateLimit-Limit", conf.count,
-                             "X-RateLimit-Remaining", remaining)
+    if conf.show_limit_quota_header == nil or conf.show_limit_quota_header then
```
Comment (spacewander) [inline apisix/plugins/limit-count.lua:188]:
```suggestion
    if conf.show_limit_quota_header then
```

The default value will be set in `check_schema` even the configuration in etcd doesn't have it.

Comment (okaybase) [reply apisix/plugins/limit-count.lua]: okay, done~

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**